ui/src/component/ClientGoal.vue

```diff
@@ -86,17 +85,28 @@ export default {
   methods: {
     async updateNotes() {
       try {
+        const payload = {
+          health_note: this.notes.healthMedsNote,
+          goal_note: this.notes.goalsNote,
+          misc_note: this.notes.miscNote,
+        };
+
+        if (
+          this.notes.goalWeight !== undefined &&
+          this.notes.goalWeight !== ""
```

Comment:
you can do 
```if (this.notes.goalWeight) {}```
to cover all the empty  cases

Reply:
fixed